packages/docs/site/docs/10-javascript-api/03-playground-api-client.md

```diff
@@ -70,6 +70,44 @@ await client.goTo('/test/index.php');
 
 For a full list of these methods, consult the PlaygroundClient interface.
 
+## Sending messages to JavaScript
+
+You can pass messages from PHP to JavaScript using the `post_message_to_js()` function. It accepts two arguments:
+
+-   `$type` – is used to call the correct listener
+-   `$data` – any extra information as a string
+
+For example, here's how you would send a message called "post-save" with a JSON-encoded post ID and title:
+
+```ts
+const php = await PHP.load('8.0');
+
+php.onMessage(
+	// The data is always passed as a string
+	function (data: string) {
+		// Let's decode and log the data:
+		console.log(JSON.parse(data));
+	}
+);
+
+// Now that we have a listener in place, let's
+// dispatch a message:
+await php.run({
+	code: `<?php
+        post_message_to_js(
+            "post-save",
```

Comment:
I don't see this identifier in the wasm source (with the caveat that I don't know wasm), so I suppose it is a leftover from an initial signature, @adamziel

Reply:
It is, great spot @mcsf!

Reply:
Fixed in ad2118fa3fa967000ca6e484e9affd806f6b4f5a